lib/msal-node/test/network/HttpClient.spec.ts

```diff
@@ -199,44 +199,48 @@ describe("HttpClient", () => {
         });
     });
 
-    describe("Bad Status Code Error", () => {
+
+    describe("Bad Status Code Error", <T>() => {
         const statusCodeError: number = 500;
-        const error: Error = new Error(`HTTP status code ${statusCodeError}`);
+        const error: Error = new Error("Error in connection to proxy");
+        const networkResponse: NetworkResponse<T> = getNetworkResponse(mockGetResponseBody, statusCodeError);
```

Comment:
Create a local type for this payload:

```typescript
export type MockedMetadataResponse = {
    tenant_discovery_endpoint: string,
    token_endpoint: string,
    authorization_endpoint: string,
    device_authorization_endpoint: string
}

Then do this:
```suggestion
        const networkResponse: NetworkResponse<MockedMetadataResponse> = getNetworkResponse<MockedMetadataResponse>(mockGetResponseBody, statusCodeError);
```

Reply:
Thanks @jasonnutter !